projects.go

```diff
@@ -108,6 +109,20 @@ type ProjectNamespace struct {
 	UpdatedAt   *time.Time `json:"updated_at"`
 }
 
+// StorageStatistics represents a statistics record for a group or project.
+type StorageStatistics struct {
+	StorageSize        uint64 `json:"storage_size"`
+	RepositorySize     uint64 `json:"repository_size"`
+	LfsObjectsSize     uint64 `json:"lfs_objects_size"`
+	BuildArtifactsSize uint64 `json:"build_artifacts_size"`
+}
+
+// ProjectStatistics represents a statistics record for a project.
+type ProjectStatistics struct {
+	StorageStatistics
+	CommitCount uint64 `json:"commit_count"`
```

Comment:
No reason to use this big a value for this field. Please just use an `int` here (represents a commit count up until 2.1 billion commits).